documentation/docs/setup_a_node.mdx

```diff
@@ -156,14 +156,14 @@ rustup update
 
 ### Download the Source
 
-Once you have installed all the required dependencies, you can start compiling the Bee Node.  To do so, you can simply clone the source code by running the following command.
+Once you have installed all the required dependencies, you can start compiling the Bee Node. To do so, you can simply clone the source code by running the following command.
 
 ```shell
 git clone https://github.com/iotaledger/bee.git --branch dev
 ```
 
 #### Compiling
```

Comment:
```suggestion
### Compiling
```

Reply:
I think the headers on lines 174 and 225 should be an h4 (####) as they seem to be a subsection on compiling.